yarpl/include/yarpl/observable/ObservableOperator.h

```diff
@@ -75,10 +76,10 @@ class ObservableOperator : public Observable<D> {
     void onSubscribe(
         Reference<yarpl::observable::Subscription> subscription) override {
       if (upstream_) {
+        DLOG(ERROR) << "attempt to subscribe twice";
```

Comment:
I believe this error is just to avoid misusing the API. Using cancelled subscription for onSubscribe is as wrong as re-subscribing with already closed Subscriber. I would ideally just std::terminate for this case, but we should avoid crashing if possible.